src/Message/ApplePayAuthorizeRequest.php

```diff
@@ -0,0 +1,462 @@
+<?php
+
+namespace Omnipay\Vindicia\Message;
+
+use Omnipay\Common\Message\ResponseInterface;
+use Guzzle\Common\Event;
+
+/**
+ * Retrieve an Apple Pay Payment Session object.
+ *
+ * Post a request to Apple Pay server's Payment Session endpoint using two-way TLS.
+ * The response to this request is an Apple Pay session object – it expires after five minutes.
+ * Pass the session object to the completion function completeMerchantValidation in the front end
+ * to verify the merchant.
+ *
+ * Parameters set by the gateway:
+ * - pemCertPath: Path to the cert of the split Merchant Identity certification, associated with your
+ * merchantID needed to make the call.
+ * - keyCertPath: Path to the key cert of the split Merchant Identity certification, associated with
+ * your merchantID needed to make the call.
+ * - keyCertPassword: Key certification password for the key certification.
+ *
+ * Parameters set by this request:
+ * - validationUrl: Validation URL received from session.onvalidatemerchant on the client.
+ * - merchantIdentifier: Your merchant ID provided by Apple.
+ * - displayName: A name for your store, suitable for display, appears in the touch bar.
+ * - applicationType: The "initiative" parameter depending on the type of application:
+ *   For Apple Pay on the web, use "web".
+ * - applicationUrl: The initiative context is based on the value supplied for the applicationType/initiative:
+ *   Provide your fully qualified domain name associated with your Apple Pay Merchant Identity Certificate.
+ *
+ * See @link for more details.
```

Comment:
what's this referencing?

Reply:
This link tag in the code:

https://github.com/vimeo/omnipay-vindicia/pull/42/files#diff-ba17822789aa8e485b85ea6429963389R88

Reply:
oh maybe just move those four lines up here? might be more clear